### PR TITLE
crystal0523-proposal-attempt1

### PR DESCRIPTION
**Pull request by crystal0523**
This is my rough idea about term project, and I would like to have some idea/advice from Professor!! Any suggestion would be wonderful, thanks.

---
A good PR should include the following parts:

- [x] A clear title
- [x] A meaningful message for PR, as well as its commits
- [x] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [x] excluding any irrelevant files, such as binaries, text files, or dot files
- [x] passing tests/CI

### CODE REVIEW

File: proposal/crystal0523/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Basic Information
+* GitHub repository: https://github.com/crystal0523/nsdhw_21au/tree/crystal0523-proposal-submission/proposal/crystal0523
+## Problem to Solve
+
+The Fourier Transform is used in a wide range of image applications, which transform the signal from time domain to frequency domain, such as image analysis, image filtering, image reconstruction and image compression.
+
+The formula of 2-D Discrete Fourier Transform for an M*N image:
+> $\dfrac{1}{M*N}\sum_{x=0}^{M-1}\sum_{y=0}^{N-1}f(x,y)e^{-i2\pi[\dfrac{ux}{M}+\dfrac{vy}{N}]}$, for u = 0..M-1, v = 0..N-1
+## Prospective Users
+People who would like to do some image processing tricks.
```
Comment (yungyuc) [inline proposal/crystal0523/README.md:11]: What tricks?  Spelling out concrete tasks may help focus your efforts and project progress.

File: proposal/crystal0523/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Fourier Transform for Image Processing
+## Basic Information
+* GitHub repository: https://github.com/crystal0523/nsdhw_21au/tree/crystal0523-proposal-submission/proposal/crystal0523
```
Comment (yungyuc) [inline proposal/crystal0523/README.md:3]: You need a repository to house your code.  Fork of the homework repository cannot be used.

File: proposal/crystal0523/README.md (hunk 1 above)
Comment (yungyuc) [reply proposal/crystal0523/README.md]: Identifying at least two processing techniques is a good progress.  You have moved forward from nothing is clear to something is clear.  It is OK to have something unclear at this moment.  Good job.

File: proposal/crystal0523/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Fourier Transform for Image Processing
+## Basic Information
+* GitHub repository: https://github.com/crystal0523/nsdhw_21au
```
Comment (yungyuc) [inline proposal/crystal0523/README.md:3]: This is a fork of the homework repository and cannot be used for your project repository.  Please create a new repository dedicated for your project.

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: proposal/crystal0523/README.md (hunk 3 above)
Comment (crystal0523) [reply proposal/crystal0523/README.md]: Got it, thanks for reminding me.

**Review by yungyuc — APPROVED** [review]

**Pull request merged**